src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/GregtechMetaTileEntity_IndustrialChisel.java

```diff
@@ -66,7 +66,7 @@ protected GT_Multiblock_Tooltip_Builder createTooltip() {
 				.addInfo("Factory Grade Auto Chisel")
 				.addInfo("Target block goes in GUI slot")
 				.addInfo("If no target provided, firdt chisel result is used")
-				.addInfo("Speed: 200% | Eu Usage: 75% | Parallel: Tier x 16")
+				.addInfo("Speed: +200% | Eu Usage: 75% | Parallel: Tier x 16")
```

Comment:
Can you capitalise Eu to EU while you are here? Same for the other tooltips.

Reply:
Done.

Also I wonder if Adv DT's parrellel is related to Tower Tier in DTower Mode. It seems the parrellel is 4 * PowerInputTier either in T1 or T2. If so, tooltip needs to be updated.